pkgs/development/python-modules/untangle/default.nix

```
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "untangle";
+  version = "1.1.1";
+  
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e7cfa1ad57707e6b74cfea8b9fc50f7cbe9bbaf18401cc9d72192002bcd80bcb";
+  };
+  
```

add tests please
```suggestion
  
  checkInputs = [ pytest ];
  checkPhase = ''
    pytest tests/
  '';
```

looks like you will have to check out from github to pick up tests as well.

Tests are important to make sure the package still works even after bump it's dependencies

---

Good point, did not see that tests were simply omitted.
Let me review my other PRs as well and update accordingly.